docs/src/release-notes-python.md

```diff
@@ -5,6 +5,76 @@ title: "Release notes"
 
 <!-- TOC -->
 
+## Version 1.18
+
+### API Testing
+
+Playwright for Python 1.18 introduces new [API Testing](./api/class-apirequestcontext) that lets you send requests to the server directly from Python!
+Now you can:
+
+- test your server API
+- prepare server side state before visiting the web application in a test
+- validate server side post-conditions after running some actions in the browser
+
+To do a request on behalf of Playwright's Page, use **new [`property: Page.request`] API**:
+
+```python
+// Do a GET request on behalf of page
+res = await page.request.get("http://example.com/foo.json");
+```
+
+Read more in [our documentation](./api/class-apirequestcontext).
+
+### Web-First Assertions
+
+Playwright for Python 1.18 introduces [Web-First Assertions](./api/class-playwrightassertions).
+
+Consider the following example:
+
+```python
+from playwright.async_api import Page, expect
+
+async def test_status_becomes_submitted(page: Page) -> None:
+    # ..
+    await page.click("#submit-button")
+    await expect(page.locator(".status")).to_have_text("Submitted")
```

Comment:
the vast majority of people uses sync, so we should also provide a sync example.

Reply:
done